Doc/howto/clinic.rst

```diff
@@ -13,12 +13,19 @@ Argument Clinic How-To
 
   Argument Clinic is a preprocessor for CPython C files.
   Its purpose is to automate all the boilerplate involved
-  with writing argument parsing code for "builtins".
-  This document shows you how to convert your first C
-  function to work with Argument Clinic, and then introduces
-  some advanced topics on Argument Clinic usage.
+  with writing argument parsing code for "builtins",
+  module level functions, and class methods.
+  This document is divided in three major sections:
 
-  Currently Argument Clinic is considered internal-only
+  * :ref:`clinic-background` talks about the basic concepts and goals of
+    Argument Clinic.
+  * :ref:`clinic-tutorial`
+  * :ref:`clinic-howtos` details how to handle specific tasks.
```

Comment:
The second item of the list is missing a period, and I think it might benefit from a short description like the other two.  The diataxis website says: 
> Tutorials are lessons that take the reader by the hand through a series of steps to complete a project of some kind. Tutorials are learning-oriented.

So maybe something like this (even though it's not very short):
```suggestion
  * :ref:`clinic-background` talks about the basic concepts and goals of
    Argument Clinic.
  * :ref:`clinic-tutorial` guides you through all the steps required to
    convert a function to work with Argument Clinic.
  * :ref:`clinic-howtos` details how to handle specific tasks.
```


Side note: is it normal that the preview colors look broken?
![image](https://github.com/python/cpython/assets/25624924/631a7d47-4ae5-47f5-879c-0968152ec213)

Reply:
OT: The Sphinx dark mode issue is already reported by Hugo: https://github.com/python/python-docs-theme/issues/132

Reply:
> The second item of the list is missing a period, and I think it might benefit from a short description like the other two.

But, the title of that item is self-describing:

<img width="622" alt="Screenshot 2023-07-20 at 19 37 21" src="https://github.com/python/cpython/assets/13780613/f9345ea0-1022-439b-b55c-cfb3ae914eea">

We could either add a period, like this:

```suggestion
  * :ref:`clinic-background` talks about the basic concepts and goals of
    Argument Clinic.
  * :ref:`clinic-tutorial`.
  * :ref:`clinic-howtos` details how to handle specific tasks.
```

Or we could shorten the tutorial title to simply "Tutorial", and describe it like you proposed in the Abstract. Perhaps the latter is the best way forward; we already used that in the `sqlite3` docs.

Reply:
Resolved with c1bf9775905586b927d52502668aa5c03fd4eedc